examples/kitchensink/app-reversed/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Reversed example in frint",
   "main": "index.js",
   "scripts": {
-    "build": "../../../node_modules/.bin/webpack"
```

Comment:
please remove App specific package.json files, so that we end up with only one single `<root>/examples/kitchensink/package.json` file.